### ISSUE TITLE
Matching?

### ISSUE THREAD

**Issue by mmarmol** [awslabs/aws-lambda-redshift-loader#8]
I have on dynamodb an entry with this prefix:

s3Prefix:   analitycs-events-store/year=_/month=_/day=*

But I can figure out how to match it on lambda, It always return that has not found the pattern on dynamodb:

2015-04-13T14:16:17.567Z    a3e758a8-e1e7-11e4-8f24-6fe7d03a27ad    unable to load configuration for analitycs-events-store/year%3D2015/month%3D04/day%3D13

Any tip?

**Comment by mmarmol** [comment]
Looks like the request is getting here escaped some how. I will add a url decode to see if it fix it.  

  if (searchKey && searchKey !== null && searchKey !== "") {
                                        // transform hive style dynamic prefixes into static match prefixes
                                        do {
                                                searchKey = searchKey.replace(/(=\d+)+/, "=*");
                                        } while (searchKey.match(/(=\d+)+/) !== null);

```
                                    searchKey = "/" + searchKey;
                            }
```

**Comment by mmarmol** [comment]
inputInfo.key = decodeURIComponent(r.s3.object.key);

I had to add some decodeURI for some reason. It seems that when the event actually comes from amazon aws it comes encoded. Now It is working.

**Comment by IanMeyers** [comment]
Thanks - sorry to have missed this one. Will patch and push a 1.1.3 version very soon.

**Comment by IanMeyers** [comment]
Fixed in 1.1.4+
